Source/Layout/ASLayout.mm

```diff
@@ -76,6 +76,18 @@ @implementation ASLayout
 
 @dynamic frame, type;
 
+static std::atomic_bool static_retainsSublayoutLayoutElements = ATOMIC_VAR_INIT(NO);
```

Comment:
Added `static` prefix because `ASLayout` also has an instance variable called `retainSublayoutLayoutElements ` [here](https://github.com/TextureGroup/Texture/blob/master/Source/Layout/ASLayout.mm#L64).